.github/workflows/e2e-tests.yml

```diff
@@ -38,4 +38,4 @@ jobs:
         run: uv sync --locked --group test
 
       - name: Run test with pytest via uv
-        run: DOCKER_HOSTNAME=172.17.0.1 uv run pytest tests/e2e --build
+        run: DOCKER_HOSTNAME=172.17.0.1 TESTS_USE_RABBITMQ=true uv run pytest tests/e2e --build
```

Comment:
`TESTS_USE_RABBITMQ=true` here allows tests to trigger tasks in the worker container.